tests/unit/splinterdb_stress_test.c

```diff
@@ -132,11 +129,11 @@ CTEST2(splinterdb_stress, test_naive_range_delete)
       ASSERT_EQUAL(0, rc);
    }
 
-   platform_default_log("loaded %u k/v pairs\n", num_inserts);
+   CTEST_LOG_INFO("loaded %u k/v pairs\n", num_inserts);
 
    uint32 num_rounds = 5;
    for (uint32 round = 0; round < num_rounds; round++) {
-      platform_default_log("range delete round %d...\n", round);
+      CTEST_LOG_INFO("range delete round %d...\n", round);
```

Comment:
My comment is not specifically tied to this diff, or the diff below on L164.

I just wondered ... how would a new test author know which interface to use for printing messages.

Unit-tests: I noticed that in `splinter_test.c`, we are still using `platform_default_log()`, while in other files we are using `CTEST_LOG_INFO()`.

Functional-tests: are all using `platform_default_log()`.

I am wondering ... how can we provide guidelines on what to use where. May be a `README` in `tests/` sub-dir with some general guidelines on what to use where (`platform_default_log()`, `CTEST_LOG_INFO()` and `CTEST_ERR()`) will be helpful. I will certainly forget these details and will benefit from archived how-to instructions for future reference.

Reply:
Yes, that was a problem before this PR (multiple ways to print between tests).  I think `splinter_test.c` is a special case, because it is testing private APIs.  That said, based on this feedback, I've made the following updates to this PR:
1. updated it to use the `CTEST_LOG_INFO()` macro as much as I could easily... though there are other places where it uses private APIs to print to the `Platform_default_log_stream` and I'm not going to touch that in this PR.  
2. Made it respect the verbose flag by calling `platform_set_log_streams()` the right way
3. added a note to the top of that test file now to warn folks not to follow patterns established there.